moveit_ros/planning/moveit_cpp/include/moveit/moveit_cpp/planning_component.h

```diff
@@ -202,10 +202,10 @@ class PlanningComponent
 
   /** \brief Execute the latest computed solution trajectory computed by plan(). By default this function terminates
    * after the execution is complete. The execution can be run in background by setting blocking to false. */
-  [[deprecated("Use MoveItCpp::execute()")]] bool execute(bool blocking = true);
-
-  /** \brief Return the last plan solution*/
-  const planning_interface::MotionPlanResponse& getLastMotionPlanResponse();
```

Comment:
@sjahr I think you might have screwed up the previous commit. Didn't you mean to add a deprecation warning here?

Reply:
I'll fix that

Reply:
:thinking: After giving it a second thought, I am unsure how I can deprecate this function in a good way. It returns a reference to a not existing member variable. If I'd keep the member variable the whole point of the PR is gone. @AndyZe Should we keep it this way? It is either way a breaking change

Reply:
I see. Well, let's just delete it then. The nice thing about Rolling is we don't have to worry too much about API breakage  :+1: